src/tigerbeetle/benchmark_load.zig

```diff
@@ -129,7 +129,15 @@ pub fn main(
         );
     }
 
-    var rng = std.rand.DefaultPrng.init(42);
+    const seed: usize = cli_args.seed orelse seed: {
+        // If no seed was given, use a random seed instead.
+        var seed_random: u64 = undefined;
+        try std.os.getrandom(std.mem.asBytes(&seed_random));
+        break :seed seed_random;
+    };
```

Comment:
Do you want to do it here or in another PR?

```suggestion
    // If no seed was given, use a default seed for reproducibility.
    const seed: usize = cli_args.seed orelse 42;
```

Reply:
I'll do it here - I'll commit and rebase this once I'm finished digging into my strange results!

Reply:
@batiati done!